package.json

```diff
@@ -2,13 +2,14 @@
   "name": "wo-ist-markt",
   "version": "0.0.1",
   "devDependencies": {
+    "colors": "~1.1.2",
     "jshint": "~2.9.1",
     "moment": "~2.11.0",
     "opening_hours": "~3.4.0"
   },
   "scripts": {
-      "lint": "jshint js preprocessing",
-      "test": "npm run lint"
+      "lint": "jshint js preprocessing validation",
+      "test": "npm run lint; node validation/markets-validator.js"
```

Comment:
Can you please add a separate `validate` command:
```
"scripts": {
    "lint": "jshint js preprocessing validation",
    "validate": "node validation/markets-validator.js",
    "test: "npm run lint; npm runt validate"
}
```
That way validation can be run without linting.